packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -51,50 +60,82 @@ contract SuperchainConfig is Initializable, ISemver {
 
     /// @notice Initializer.
     /// @param _guardian    Address of the guardian, can pause the OptimismPortal.
-    /// @param _paused      Initial paused status.
-    function initialize(address _guardian, bool _paused) external initializer {
+    /// @param _pauseExpiry Duration in seconds after which a pause expires.
+    function initialize(address _guardian, uint256 _pauseExpiry) external initializer {
```

Comment:
Let's write an explicit comment somewhere that points out the fact that if the SuperchainConfig is paused before the upgrade then it will not be paused after the upgrade. Therefore, you should not perform this upgrade while the system is paused.